modules/030-cloud-provider-yandex/openapi/config-values.yaml

```diff
@@ -1,5 +1,23 @@
 type: object
 properties:
+  cloudMetricsExporterEnabled:
```

Comment:
```yaml
cloudMetrics:
  exporterEnabled: true | false
  apiKey : "abcdef"
```

Reply:
Like here
https://github.com/deckhouse/deckhouse/blob/be7243da49f8c7f47ca03f338ac44439a9dd8afe/ee/fe/modules/340-extended-monitoring/openapi/config-values.yaml#L3-L54

Reply:
With `@konstantin-axenov` we decided use one parameter in withNatInstance layout config for apiKey.